machine/v1/types_controlplanemachineset.go

```diff
@@ -287,6 +287,14 @@ type AzureFailureDomain struct {
 	// If nil, the virtual machine should be deployed to no zone.
 	// +kubebuilder:validation:Required
 	Zone string `json:"zone"`
+
+	// Subnet for the virtual machine.
```

Comment:
Comment should start with the json tag name. Can we expand on this any more?
```suggestion
	// subnet is the name of the network subnet in which the VM will be created.
	// When omitted, the subnet value from the machine providerSpec template will be used.
```